content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -83,10 +83,46 @@ a different label key if you prefer.
 
 ## Automatic node labelling {#node-labeller}
 
-If you're using AMD GPU devices, you can deploy
-[Node Labeller](https://github.com/RadeonOpenCompute/k8s-device-plugin/tree/master/cmd/k8s-node-labeller).
-Node Labeller is a {{< glossary_tooltip text="controller" term_id="controller" >}} that automatically
-labels your nodes with GPU device properties.
+As an administrator, you can automatically discover and label all your GPU enabled nodes
+by deploying Kubernetes [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) (NFD).
+NFD detects the hardware features that are available on each node in a Kubernetes cluster.
+Typically, NFD is configured to advertise those features as node labels, but NFD can also add extended resources, annotations, and node taints.
+NFD is compatible with all [supported versions](/releases/version-skew-policy/#supported-versions) of Kubernetes.
 
```

Comment:
```suggestion
You also need a plugin for NFD that adds labels to your nodes. Your GPU vendor may provide a third party
plugin for NFD; check their documentation for more details.

```

Reply:
I did some mods to this suggestion please check

Reply:
I did not find any mentions of "NFD plugin" from NFD documentation:  https://kubernetes-sigs.github.io/node-feature-discovery

Whereas NFD release notes mentioned "NFD plugin" to be functionality that NFD provides for `kubelet`.

If "NFD plugin" in above means those "hook" binaries that NFD worker called earlier, that support is disabled by default, and will be removed in next NFD release (due to security reasons), so I think some better term could be used.

Something like this sounds a bit too much though:
> NFD alone does not label nodes, example here requires additional vendor support, either NFD configuration specifying device matching rules it will use for labeling, or additional component directly providing NFD node worker list of labels.